src/Markdown.ts

```diff
@@ -15,16 +16,16 @@ limitations under the License.
 */
 
 import * as commonmark from 'commonmark';
```

Comment:
If you install https://www.npmjs.com/package/@types/commonmark we'll get access to all its goodies

Reply:
Ah, it *is* a thing. We still have to expose the methods for the individual types of element, but it's better.